CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contributing to HiGlass
+
+🎉 Hooray, it's marvelous to see you here! 🥳 Thanks for considering to contribute to HiGlass. 👍
```

Comment:
Odd unicode character?

![image](https://user-images.githubusercontent.com/2143629/61681946-79e30780-acc4-11e9-80e3-654f0fb0b83d.png)

Reply:
<img width="141" alt="Screen Shot 2019-07-23 at 12 23 05 AM" src="https://user-images.githubusercontent.com/932103/61682539-12d34c00-ace0-11e9-9c79-ef455c219c14.png">

I can remove the emoji :)